packages/helix/build.sh

```
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -ex
+export CC=gcc
+export LD=gcc
+export RUSTFLAGS="-C linker=gcc"
+export HELIX_DEFAULT_RUNTIME=/usr/lib/helix/runtime
+export HELIX_DISABLE_AUTO_GRAMMAR_BUILD=1
+
+cargo build --release
+
+mkdir -p $OUTPUT_DIR/usr/bin
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)